src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/StreamHelper.cs

```diff
@@ -309,6 +307,9 @@ internal static void WriteToStream(Stream input, Stream output, PSCmdlet cmdlet,
             catch (OperationCanceledException)
             {
             }
+            catch (AggregateException)
+            {
```

Comment:
I think that when we take this change, this AggregateException catch should follow the common pattern and unwrap the collected exceptions.
```suggestion
            {
                ae.Handle((x) =>
                {
                    return true;  // Let all wrapped exceptions throw and be handled in calling layers
                }
            }
```